src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -5142,6 +5143,29 @@ .maxstack  7
 ");
         }
 
+        [Theory, WorkItem(48919, "https://github.com/dotnet/roslyn/issues/48919")]
```

Comment:
One part that I don't see specified in the notes is what happens when the pointer is cast to `void*` before comparison. My assumption is that this would not warn (customer has hidden the function pointer from us hence it's just a `void*` comparison). If that's the case think we should have a copy of this test where the operands are cast to `void*` and verify it does not warn.

Reply:
Yeah, if you explicitly cast to `void*` there is no warning. I'll add a test.